src/main/kotlin/no/nav/amt/deltaker/bff/deltakerliste/DeltakerlisteMedArrangor.kt

```diff
@@ -0,0 +1,8 @@
+package no.nav.amt.deltaker.bff.deltakerliste
+
+import no.nav.amt.deltaker.bff.arrangor.Arrangor
+
+data class DeltakerlisteMedArrangor(
```

Comment:
Trenger ikke å endre dette for min del nå men:
Hvis vi stortsett alltid tror vi trenger arrangør sammen med deltakerlisten, kan vi gjerne legge arrangør inn i `Deltakerliste` i steden for bare å ha id der.

Jeg blir for forvirret med mange variasjoner av det som egentlig er samme klasse 😅

Reply:
Det kan vi veldig gjerne gjøre. Skal jeg bare gjøre det med en gang?

Reply:
Gjerne det 👍